src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss

```diff
@@ -2,31 +2,40 @@
 
 .ScheduleItem {
   display: flex;
-  padding: 10px 16px;
-  margin-top: 10px;
-  margin: 10px -16px 0px -16px;
+  padding: 10px 0;
```

Comment:
Maybe extract constants over this file? But if it's from an old schedule and is eventually be removed then never mind.

Reply:
Yes, it's from the old schedule.